Watchman.Discord/WatchmanBot.cs

```diff
@@ -64,10 +69,12 @@ private void LogException(Exception e, SocketMessage socketMessage)
                 messagesService.SendMessage("Wystąpił wyjątek");
             }
 #if DEBUG
-            messagesService.SendMessage($"```Command: {socketMessage.Content}```");
-            messagesService.SendMessage($"```Message: {e.Message}```");
-            messagesService.SendMessage($"```InnerException message: {e.InnerException?.Message}```");
-            messagesService.SendMessage($"```InnerException2 message: {e.InnerException?.InnerException?.Message}```");
+            var exceptionMessageBuilder = new StringBuilder();
```

Comment:
Zrób extension method który przyjmuje IEnumerable<string> i string[] (polimorfizm) 
i tworzy tekst w stylu z ``` ``` => może użyć obecnych metod do tworzenia takiego tekstu 

aktualnie ten kod nie wygląda za bardzo atrakcyjnie 

tak samo przyda sie ExtensionMethod do Exception
które zmienia to na stringa, gotowego do wyświetlenia 
możesz pomyśleć nad ogarnianiem Exceptions w ResponsesService => może powinno być odpowiednie response dla wyjątków, gdzie możesz sam zdefiniować gdzie wyświetlasz {{message}} gdzie {{innerMessage}} gdzie {{stackTrace}} itd